model/Export/Stylesheet/AssetStylesheetLoader.php

```diff
@@ -41,13 +42,25 @@ public function loadAssetFromAssetResource(string $link): ?StreamInterface
         $asset = $this->getResource(UriHelper::decode($link));
 
         if ($asset->exists()) {
-            $property = (string) $asset->getUniquePropertyValue(
+            $property = (string)$asset->getUniquePropertyValue(
                 $this->getProperty(AbstractQTIItemExporter::PROPERTY_LINK)
             );
 
-            return $this->getFileManagement()->getFileStream(
-                $this->buildAssetPathFromPropertyName($property)
-            );
+            $stylesheetPath = $this->buildAssetPathFromPropertyName($property);
+            try {
+                return $this->getFileManagement()->getFileStream(
+                    $stylesheetPath
+                );
+            } catch (FileNotFoundException $exception) {
```

Comment:
are you sure that is the error we are going to get ? 
should we not check for throwable  and store the error ? 
btw , why are you silencing the error and what is the side effects of this ?

Reply:
Agreed with @mike-ionut-mihai-sandu-tao 
Please, use `Throwable` interface to catch all possible issues instead of `FileNotFoundException` or in an additional `catch` block:
<details>
<summary>Option 1</summary>

```php
try {
    ... code ...
} catch (Throwable $exception) {
    ... code ...
}
```
</details>
<details>
<summary>Option 2</summary>

```php
try {
    ... code ...
} catch (FileNotfoundException $exception) {
    ... code ...
} catch (Throwable $exception) {
    ... code ...
}
```
</details>
<details>
<summary>Option 3</summary>

```php
try {
    ... code ...
} catch (FileNotfoundException | Throwable $exception) {
    ... code ...
}
```
</details>